modules/govuk_gor/manifests/init.pp

```diff
@@ -47,6 +47,20 @@
     require => Class['govuk::deploy'],
   }
 
+  cron { 'stop_gor':
+    command => 'echo "true" > /etc/govuk/env.d/FACTER_data_sync_in_progress; sudo initctl stop gor',
+    user    => $govuk_env_sync::user,
```

Comment:
As this isn't directly associated with the env sync stuff, that user probably shouldn't be used. I would be unsure if that user even exists on the cache machines.

Given you're writing to the `/etc/govuk/env.d/FACTER_data_sync_in_progress` file, maybe consider using the `deploy` user?